clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -160,65 +205,301 @@ const SystemInformationForm = ({
+                        label="This system requires Data Privacy Assessments"
+                        tooltip="Does this system require (DPA/DPIA) assessments?"
+                        variant="stacked"
+                      />
+                      <Collapse
+                        in={values.requires_data_protection_assessments}
+                        animateOpacity
+                      >
+                        <Box mt={4}>
+                          <CustomTextInput
+                            label="DPIA/DPA location"
+                            name="dpi_location"
+                            tooltip="Where is the DPA/DPIA stored?"
+                            variant="stacked"
+                            isRequired={
+                              values.requires_data_protection_assessments
+                            }
+                          />
+                        </Box>
+                      </Collapse>
+                    </Stack>
+                  </Stack>
+                </Collapse>
+              </Stack>
+            </SystemFormInputGroup>
+            <Collapse
+              in={
+                values.processes_personal_data &&
+                !values.exempt_from_privacy_regulations
+              }
+              animateOpacity
+            >
+              <SystemFormInputGroup heading="Administrative properties">
                 <CustomTextInput
-                  id="name"
-                  name="name"
-                  label="System name"
-                  tooltip="Give the system a unique, and relevant name for reporting purposes. e.g. “Email Data Warehouse”"
+                  label="Data steward"
+                  name="data_steward"
+                  tooltip="Who are the stewards assigned to the system?"
                   variant="stacked"
+                  disabled={
+                    !values.processes_personal_data ||
+                    values.exempt_from_privacy_regulations
+                  }
                 />
-                {isEditing && (
-                  <CustomTextInput
-                    id="fides_key"
-                    name="fides_key"
-                    label="System Fides key"
-                    disabled
-                    tooltip="A string token of your own invention that uniquely identifies this System. It's your responsibility to ensure that the value is unique across all of your System objects. The value may only contain alphanumeric characters, underscores, and hyphens. ([A-Za-z0-9_.-])."
-                    variant="stacked"
-                  />
-                )}
                 <CustomTextInput
-                  id="description"
-                  name="description"
-                  label="System description"
-                  tooltip="If you wish you can provide a description which better explains the purpose of this system."
+                  label="Privacy policy URL"
+                  name="privacy_policy"
```

Comment:
I think we need to add/update the Yup schema to validate that this field has valid urls. Right now it's possible to submit anything and the backend returns an error.

here's an example

```typescript
import { object, string, number, date, InferType } from 'yup';

let userSchema = object({
  name: string().required(),
  age: number().required().positive().integer(),
  email: string().email(),
  website: string().url().nullable(),
  createdOn: date().default(() => new Date()),
});
```


If there isn't a validation schema for this for yet try only adding this field to keep in simple.

Reply:
It looks like this field is being treated as required by the backend. Here is the error being returned

```json
{
    "detail": [
        {
            "loc": [
                "body",
                "privacy_policy"
            ],
            "msg": "ensure this value has at least 1 characters",
            "type": "value_error.any_str.min_length",
            "ctx": {
                "limit_value": 1
            }
        }
    ]
}
```

Reply:
I can't reproduce the `min_length` error and it shows as optional in the type file, but I'll throw `.nullable()` on the validation schema anyway?